examples/futures_wp/src/markdown.rs

```diff
@@ -0,0 +1,185 @@
+/// Original author of this code is [Nathan Ringo](https://github.com/remexre)
+/// Source: https://github.com/acmumn/mentoring/blob/master/web-client/src/view/markdown.rs
```

Comment:
Do you know what the license for this file is?

Reply:
It's already in the repo, in `examples/crm/src/markdown.rs`

I figured a second copy would be harmless (?)

Reply:
That's cool! It might be better to place it in a separate crate which both of the examples can depend on to reduce code duplication?

Reply:
Done!